static/app/components/forms/rangeField.tsx

```diff
@@ -66,7 +66,7 @@ export default function RangeField({
           {...fieldProps}
           value={value}
           onBlur={onBlur}
-          onChange={(val, event) => onChange(fieldOnChange, val, event)}
+          onChange={(val, event) => onChange(fieldOnChange, event, val)}
```

Comment:
Do these arguments still need to be swapped?

Reply:
The optional parameter needs to be last